packages/dd-trace/src/plugin_manager.js

```diff
@@ -55,6 +55,10 @@ module.exports = class PluginManager {
         }
         this.configurePlugin(name, pluginConfig)
       }
+    } else {
+      for (const name in this._pluginsByName) {
+        this.configurePlugin(name, false)
+      }
```

Comment:
What does all this do?

Reply:
Its to help with the merging of the two plugins in datadog-plugin-http/src/index.js @rochdev

Reply:
Not only that, but without it it becomes impossible to disable plugins after they have been enabled.